src/index.html

```diff
@@ -11,8 +11,65 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), stylesheets with a `.scss` extension should be included as regular `.css` files in HTML. You should compile `.scss` to `.css` and then link to the `.css` file in your HTML.